styles.css

```diff
@@ -126,3 +128,226 @@
+    background: left no-repeat url(img/logo.svg);
+  }
+
+  .menu {
+    display: flex;
+    padding-left: 63%;
+  }
+
+  .menu__item {
+    margin-left: 24px;
+    font: 15px/51px roboto, helvetica, arial, sans-serif;
+  }
+
+  .menu__item_active::after {
+    display: block;
+    top: 35px;
+  }
+
+  .main {
+    margin-top: 0;
+    padding: 20px 40% 80px 0;
+    background: top right no-repeat url(img/frame1024.svg);
+    background-size: 55%;
+  }
+
+  .main__title {
+    margin: 0;
+    background-position: top;
+    font: 50px/66px pt_serifbold, arial, helvetica, sans-serif;
+  }
+
+  .main__subtitle {
+    margin-top: 40px;
+    font: 18px/20px roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    margin-top: 41px;
+    padding: 13px 16px 15px 23px;
+    font: 14px/16px roboto, arial, helvetica, sans-serif;
+  }
+}
+
+@media (max-width: 768px) {
+  .wrapper {
+    margin: auto;
+    width: 90%;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .name {
+    opacity: 0;
+  }
+
+  .header {
+    display: flex;
+    justify-content: space-between;
+    margin: 23px 0 0 0;
+  }
+
+  .main {
+    text-align: center;
+    margin-top: 32px;
+    padding: 80% 0 0;
+    background-size: 88%;
+    background-position: top center;
+  }
+
+  .main__title {
+    margin-top: 5px;
+    font: 30px/40px pt_serifbold, arial, helvetica, sans-serif;
+  }
+
+  .main__subtitle {
+    margin-top: 30px;
+    padding-left: 50px;
+    font: 14px/17px roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    margin-top: 28px;
+    padding: 13px 16px 15px 23px;
+    font: 14px/16px roboto, arial, helvetica, sans-serif;
+  }
+
+  .mobile-menu {
+    position: fixed;
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    padding: 0 16px;
+    top: 16px;
+    left: 0;
+    right: 0;
+    height: 50px;
+    z-index: 9999999;
+  }
+
+  .mobile-menu__btn {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    top: 16px;
+    right: 7px;
+    width: 30px;
+    height: 14px;
+    z-index: 999;
+    cursor: pointer;
+    transition: .4s;
+    background: #fff;
+  }
+
+  .mobile-menu__icon {
+    display: block;
+    position: relative;
+    background: #0056ec;
+    width: 80%;
+    height: 3px;
+    transition: .4s;
+  }
+
+  .mobile-menu__icon::after,
+  .mobile-menu__icon::before {
+    content: "";
+    display: block;
+    position: absolute;
+    background: #0056ec;
+    width: 100%;
+    height: 3px;
+    transition: .4s;
+  }
+
+  .mobile-menu__icon::after {
+    top: 8px;
+  }
+
+  .mobile-menu__icon::before {
+    top: -8px;
+  }
+
+  .mobile-menu__container {
+    position: fixed;
+    text-align: center;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    top: 20px;
+    left: 0;
+    right: 0;
+    z-index: 998;
+    height: 0;
+    opacity: 1;
+    transition: .5s;
+    overflow: hidden;
+    background-color: #fff;
+   }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/HTML_SNN/pullRequest?prid=6307906)